7.7.поиск_ошибок_и_ревью_кода/7.7.5.py

```diff
@@ -0,0 +1,7 @@
+s = 0
+for i in range(7):
+    n = int(input())
+    if n % 2 == 0:
+
+        s = s + n
```

Comment:
Вот тут лучше использовать расширенные операторы присваивания:
```suggestion
        s += n
```